### PR TITLE
Creator and contributor need to be added to the override

### PR DESCRIPTION
**Pull request by BertZZ**
I forgot yesterday :sweat_smile:

### CODE REVIEW

File: app/views/hyku/api/v1/work/_work.json.jbuilder [hunk 1]
```diff
@@ -14,6 +14,8 @@ json.buy_book work.try(:buy_book)
 json.challenged work.try(:challenged)
 json.cname @account.cname
 #                                         "collections" => nil,
+json.contributor work.contributor
+json.creator work.creator
```
Comment (cjcolvar) [inline app/views/hyku/api/v1/work/_work.json.jbuilder:18]: Do these need to be `contributor_display` and `creator_display` instead?  `contributor` and `creator` hold the JSON blob.

Comment (BertZZ) [reply app/views/hyku/api/v1/work/_work.json.jbuilder:18]: Oooop. Ill check

Comment (BertZZ) [reply app/views/hyku/api/v1/work/_work.json.jbuilder:18]: `creator_display` is not a method. We can do `JSON.parse(creator)`

File: app/views/hyku/api/v1/work/_work.json.jbuilder [hunk 2]
```diff
@@ -14,6 +14,15 @@ json.buy_book work.try(:buy_book)
 json.challenged work.try(:challenged)
 json.cname @account.cname
 #                                         "collections" => nil,
+creator = work.creator.try(:first)
+if creator
+  json.creator JSON.parse(creator)
+end
```
Comment (cjcolvar) [inline app/views/hyku/api/v1/work/_work.json.jbuilder:20]: For these JSON fields, do you think we need to return an empty array in the response JSON if no values exist?
```suggestion
json.creator creator.present? ? JSON.parse(creator) : []
```

**Review by cjcolvar — COMMENTED** [review]
Looks like there are some rubocop violations.

**Review by cjcolvar — APPROVED** [review]
This looks good.  I'm going to merge it now, but I wonder if there are any other JSON fields that need to have the same treatment.

**Pull request merged**